app/containers/MessageBox/index.js

```diff
@@ -445,13 +489,20 @@ class MessageBox extends Component {
 
 	getEmojis = (keyword) => {
 		if (keyword) {
-			this.customEmojis = database.objects('customEmojis').filtered('name CONTAINS[c] $0', keyword).slice(0, 4);
-			this.emojis = emojis.filter(emoji => emoji.indexOf(keyword) !== -1).slice(0, 4);
+			this.customEmojis = database.objects('customEmojis').filtered('name CONTAINS[c] $0', keyword).slice(0, MENTIONS_COUNT_TO_DISPLAY);
+			this.emojis = emojis.filter(emoji => emoji.indexOf(keyword) !== -1).slice(0, MENTIONS_COUNT_TO_DISPLAY);
```

Comment:
change `emoji.indexOf(keyword) !== -1)` to `emoji.includes(keyword)`

Reply:
thats legacy code I don't wrote that , let stay it that way